.travis.yml

```diff
@@ -7,7 +7,6 @@ env:
     # We want to run both (unit) tests and end to end tests as a matrix
     # https://docs.travis-ci.com/user/speeding-up-the-build/#Parallelizing-your-builds-across-virtual-machines
     - TEST_SUITE=test
-    - TEST_SUITE=test-e2e
```

Comment:
The reason I removed this line was, having an e2e test for such API based site is going to be waste of time, as we know it just wont work. SO, I guess I'll better remove those flipkart-uris-test.js as well as this line too. Which means, no e2e test for FK!